packages/shared/src/components/Feed.tsx

```diff
@@ -72,6 +73,10 @@ const SharePostModal = dynamic(
 const PostModal = dynamic(
   () => import(/* webpackChunkName: "postModal" */ './modals/PostModal'),
 );
+const SquadPostModal = dynamic(
```

Comment:
This is actually a `SharePostModal`, not blocker but a suggestion to change naming

Reply:
done